### PR TITLE
FIX error if image is missing

### PR DESCRIPTION
**Pull request by xini**

### CODE REVIEW

**Review by tractorcow — APPROVED** [review]

**Comment by tractorcow** [comment]
Please wait until we merge before you release a tag :) Good fix thanks.

**Comment by xini** [comment]
Sorry. It didn't let me push directly, so I didn't realise it accepted the tag, but not the commit on push. Weird. Sorry.

**Pull request merged**
